arxiv/avoiding_catastrophe/README.md

```diff
@@ -0,0 +1,10 @@
+# Avoiding Catastrophe: Active Dendrites Enable Multi-task Learning in Dynamic Environments
+
+The code for the multi-task reinforcement learning (multi-task RL) portion of this project can be found at:
+> [Sources][1]
+
+The code for the continual learning (on permutedMNIST) portion of this project can be found at:
+> [Sources][2]
+
+[1]: https://github.com/numenta/nupic.embodied/tree/frontiers/projects/multitask
+[2]: https://github.com/numenta/htmpapers/tree/master/biorxiv/going_beyond_the_point_neuron
```

Comment:
There are some figures which were included in the "Avoid Catastrophe" arXiv paper, but NOT in the in the bioRxiv paper. I think it would be better to include the following link instead of htmpaper, since it's more up-to-date with these new figures.

https://github.com/numenta/nupic.research/tree/master/projects/dendrites

Reply:
Specifically, these new figures are in the `figures.ipynb` notebook.

Reply:
Will do, thanks

Reply:
Should we publish the figures code for the dendrite segment activations too? 

![image](https://user-images.githubusercontent.com/7843162/148304889-fe13f3b7-4660-4df3-ab4b-966ef531ae6b.png)

Reply:
No, it's fine for now.